backend/v3/lumens.js

```diff
@@ -77,5 +87,5 @@ function updateApiLumens() {
     });
 }
 
-setInterval(updateApiLumens, 10 * 60 * 1000);
+setInterval(updateApiLumens, 1 * 60 * 1000);
```

Comment:
changed this to update every minute for faster debugging. Once confirmed this fix works I'll change back to every 10 minutes